packages/react/src/components/Modal/Modal.base.tsx

```diff
@@ -454,12 +454,14 @@ export const ModalBase: React.FunctionComponent<IModalProps> = React.forwardRef<
         <Layer ref={mergedRef} {...mergedLayerProps}>
           <Popup
             role={isAlertRole ? 'alertdialog' : 'dialog'}
-            aria-modal={!isModeless}
             ariaLabelledBy={titleAriaId}
             ariaDescribedBy={subtitleAriaId}
             onDismiss={onDismiss}
             shouldRestoreFocus={!ignoreExternalFocusing}
+            // Modeless modals shouldn't hide siblings.
```

Comment:
I hate that we have "modeless modals".

Not an issue with this comment in particular. I just hate it in general 😂.

Reply:
That was my thought as well... :(